stdlib/runtime/equal.gr

```diff
@@ -13,10 +13,12 @@ from WasmI32 use {
   ltS as (<),
   remS as (%),
   shl as (<<),
+  shrU as (>>),
 }
 include "runtime/unsafe/wasmi64" as WasmI64
 include "runtime/unsafe/tags" as Tags
 include "runtime/numbers" as Numbers
+include "runtime/exception"
```

Comment:
Why did these additions need to happen?

Reply:
I think I was trying to raise an artificial exception for debugging purposes earlier but forgot to remove these, mb